backend/wmg/data/transform.py

```diff
@@ -1,16 +1,86 @@
 from typing import Dict
+import tiledb
+import pandas as pd
+
+from backend.wmg.data.snapshot import CELL_TYPE_ORDERINGS_FILENAME
 
 
 def get_cells_by_tissue_type(corpus_group: str) -> Dict:
     """
     Return a list of all associated cell type ontologies for each tissue contained in the
     provided corpus
     """
-    pass
+    with tiledb.open(f"{corpus_group}/obs", "r") as obs:
+        cell_tissue_types = (
+            obs.query(attrs=[], dims=["tissue_ontology_term_id", "cell_type_ontology_term_id"])
+            .df[:]
+            .drop_duplicates()
+            .sort_values(by="tissue_ontology_term_id")
+        )
+    unique_tissue_ontology_term_id = cell_tissue_types.tissue_ontology_term_id.unique()
+    cell_type_by_tissue = {}
+    for x in unique_tissue_ontology_term_id:
+        cell_type_by_tissue[x] = cell_tissue_types.loc[
+            cell_tissue_types["tissue_ontology_term_id"] == x, "cell_type_ontology_term_id"
+        ]
+
+    return cell_type_by_tissue
 
 
-def generate_cell_ordering(cell_type_by_tissue: Dict) -> None:
+def generate_cell_ordering(snapshot_path: str, cell_type_by_tissue: Dict) -> None:
```

Comment:
nit: would have this return the DataFrame, and have the caller do the write, avoiding need for `snapshot_path`; the caller handles all file-related ops.